texttospeech/cloud-client/audio_profile.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Google Cloud Text-To-Speech API sample application for audio profile.
+
+Example usage:
+    python audio_profile.py --text "hello" --effects_profile_id
+        "telephony-class-application"
+"""
+
+import argparse
+
+from google.cloud import texttospeech_v1beta1 as texttospeech
+
+
+# [START tts_synthesize_text]
+def synthesize_text(text, output, effects_profile_id):
+    """Synthesizes speech from the input string of text."""
+    client = texttospeech.TextToSpeechClient()
+
+    input_text = texttospeech.types.SynthesisInput(text=text)
```

Comment:
Does this also support ssml? 

@dizcology Should there be a 2nd snippet for ssml support?